app/services/sort_collection_ordinally.rb

```diff
@@ -5,7 +5,7 @@
 class SortCollectionOrdinally
   include Service
 
-  def call(collection)
+  def call(collection:, column_name: 'name')
     first_sort = collection.sort_by { |e| ActiveSupport::Inflector.transliterate(e.name.downcase) }

```

Comment:
Whoops - what we need to do here is take the string passed as the column_name and use `send` to get the value.  So something like `e.send(:column_name) throughout.

Reply:
Hi @rlgreen91 , its not clear to me what you meant

Reply:
So, right now we keep calling `e.name` right?  The issue is that we don't know what the column name is going to be.  We need to basically turn it into a method that is applied to an object.  One way you can do that is with the `send` method - pass it a value and it will apply it as a method to whatever is calling it.  So this basically says take the variable `e` and apply the value `:column_name` as a method.  

Hmm, you might need to assign column name variable first, something like `:column_name = column_name`